### ISSUE TITLE
Suggest more examples on SILE homepage

### ISSUE THREAD

**Issue by shreevatsa** [simoncozens/sile#438]
This is my first time looking at SILE, and I am excited, because of this line in the manual:

> it contains a slavish port of the TeX line-breaking algorithm which has been tested to produce exactly the same output as TeX given equivalent input

(and more importantly I even see the box-glue-penalty model with similar parameters like penalty=-10000).

But what are some purposes that SILE has been used for (other than by its author), or examples of what SILE _could_ be used for? I wasn't able to satisfy my curiosity about this from the homepage. Looking around in the Github repo, I was able to find a few examples at https://github.com/simoncozens/sile/tree/master/examples . For example, this one is quite interesting to me: https://github.com/simoncozens/sile/blob/master/examples/parallel.pdf

Most people grasp things best by examples, so my humble suggestion would be to put as many examples as possible (images and SILE source code) on the homepage. It might attract more users…

**Comment by shreevatsa** [comment]
(BTW it seems that sile-typesetter.org is served via Github pages. Note that Github now lets you serve pages from a `docs/` directory in the master branch (instead a separate `gh-pages` branch), so this should be even easier to do than it would have been earlier…)

**Comment by simoncozens** [comment]
Nice, I did not know what - thank you! I will try to work on this.

In general book design *isn't* very showy and that's kind of the point; I think that if the layout engine is drawing attention to itself, it's failing. So for me boring examples are good examples! I'm not actually very good at coming up with examples that might draw in users.

But I agree that the parallel (and even the triglot) examples are compelling and I'll put them up. If people can suggest other compelling examples, I'll be glad to add them.

**Comment by alerque** [comment]
I agree more examples would be a good thing, but exhibiting specific features and full spectrum ones that show how it can be all brought together. For example book layouts may be boring but having an example of everything pulled together could be good.

On the website note, I'm not a big fan of hosting website code inside the code repository. Git offers advantages to granular repository layouts that get lost whet you just shovel everything under one roof. The `/docs` folder thing could work if we were offering Markdown formatted API documentation (such as a replacement for `documentation/sile.sil`) that should track the versions (so that when you checkout a  version of the source you get matching docs. Github's hosting can be useful for rendering something like that, but I don't think the SILE website fits this model. It should be developed independently from SILE itself.

If having access to it while still having a SILE development branch checked out is an issue, `git  worktree` can check out a branch into a folder. There should already be no excuse for it being "hard" to do on account of being in an independent branch.

**Comment by the42** [comment]
related  sile-typesetter/sile#343
